tuned/profiles/profile.py

```
@@ -1,24 +1,27 @@
+import hashlib
+import json
 
 class Profile(object):
 	"""
 	Representation of a tuning profile.
 	"""
 
-	__slots__ = ["_name", "_options", "_variables", "_units"]
+	__slots__ = ["_name", "_options", "_variables", "_units", "_base_hash"]
 
 	def __init__(self, name=None, config={}):
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)